src/bigquery.ts

```diff
@@ -376,6 +393,13 @@ export class BigQuery extends common.Service {
     });
   }
 
+  private static sanitizeEndpoint(url: string) {
```

Comment:
Could you please add a `describe` block to test this function individually in test/bigquery.ts? You can probably just copy over the tests from Storage, now that we're modeled after those functions.